### ISSUE TITLE
s3-edit.rb: "formulae require at least a URL" error

### ISSUE THREAD

**Issue by milosivanovic** [tsub/s3-edit#21]
I noticed this originally when running `brew cleanup`:
```
milos@air:~$ brew cleanup
Error: formulae require at least a URL
Please report this issue:
  https://docs.brew.sh/Troubleshooting
/opt/homebrew/Library/Homebrew/formula.rb:274:in `determine_active_spec'
/opt/homebrew/Library/Homebrew/formula.rb:209:in `initialize'
/opt/homebrew/Library/Homebrew/formulary.rb:181:in `new'
/opt/homebrew/Library/Homebrew/formulary.rb:181:in `get_formula'
/opt/homebrew/Library/Homebrew/formulary.rb:340:in `get_formula'
/opt/homebrew/Library/Homebrew/formulary.rb:418:in `factory'
/opt/homebrew/Library/Homebrew/formulary.rb:484:in `from_keg'
/opt/homebrew/Library/Homebrew/formulary.rb:457:in `from_rack'
/opt/homebrew/Library/Homebrew/cleanup.rb:85:in `stale_formula?'
/opt/homebrew/Library/Homebrew/cleanup.rb:58:in `stale?'
/opt/homebrew/Library/Homebrew/cleanup.rb:356:in `block in cleanup_cache'
/opt/homebrew/Library/Homebrew/cleanup.rb:339:in `each'
/opt/homebrew/Library/Homebrew/cleanup.rb:339:in `cleanup_cache'
/opt/homebrew/Library/Homebrew/cleanup.rb:222:in `clean!'
/opt/homebrew/Library/Homebrew/cmd/cleanup.rb:58:in `cleanup'
/opt/homebrew/Library/Homebrew/brew.rb:110:in `<main>'
```

I ran `brew readall` to identify the source of the error. It's caused by s3-edit:
```
milos@air:~$ brew readall
Error: Invalid formula: /opt/homebrew/Library/Taps/tsub/homebrew-s3-edit/s3-edit.rb
formulae require at least a URL
```

**Comment by tsub** [comment]
@milosivanovic Is the machine you are using an M1 Mac?

I found a similar issue https://github.com/goreleaser/goreleaser/issues/2389 with goreleaser, which I use to release Homebrew formulae.
According to this, it seems to reproduce on M1 Mac.

My machine is also an M1 Mac, and the problem was reproduced as well.

```
$ arch
arm64

$ brew readall
Error: Invalid formula: /opt/homebrew/Library/Taps/tsub/homebrew-s3-edit/s3-edit.rb
formulae require at least a URL
```

If you are using an M1 Mac, then supporting the binary offering for M1 Macs seems to solve this problem.

**Comment by milosivanovic** [comment]
@tsub Thanks for the quick response. Yes, I am using M1.

I don't understand what you mean by "supporting the binary offering for M1 Macs". What are the steps to resolve the issue?

**Comment by tsub** [comment]
> I don't understand what you mean by "supporting the binary offering for M1 Macs". What are the steps to resolve the issue?

@milosivanovic I'm sorry for the confusion.

There is no action for you to take on this issue at the moment.

It will be solved naturally if I write a configuration to build binaries for darwin_arm64 in `.goreleaser.yml` and release a new version.

Please wait for a while, I will mention to you when the problem is solved.

**Comment by tsub** [comment]
@milosivanovic The cause was a little different, but it was fixed in [v0.0.15](https://github.com/tsub/s3-edit/releases/tag/v0.0.15).

Please run `$ brew update` and I hope it will fix the problem.

**Comment by milosivanovic** [comment]
@tsub Oh, no need to apologise. Thanks so much for fixing, it works now! :-)
